### DISCUSSION TITLE
OpenSSL on bare metal Arm

### DISCUSSION THREAD

**Discussion by DragonsAshes** [openssl/openssl#23501] [Q&A]
Hello,
I would like to know if it is possible to compile openssl for arm bare metal ?

**Comment by tom-cosgrove-arm** [comment]
What exactly are you trying to do? Do you want to (for example) perform AES or SHA2 computations? Or do you want to connect to an HTTPS server?

**Reply by DragonsAshes** [reply]
I would like to do signature verification
